monument/cli/src/music/mod.rs

```diff
@@ -20,7 +20,9 @@ pub enum BaseMusic {
     None,
     /// A default music profile is used
     Default,
-    // TODO: Other music profiles, e.g. CompLib's
+    /// Complib music profile is used
+    Complib,
+    // TODO: Other music profiles
```

Comment:
I reckon this `TODO` comment can just be removed - CompLib music scoring was the only thing I was thinking about when I wrote the comment.